brushfire-features/src/main/scala/com/stripe/brushfire/features/DispatchedFeatureEncoder.scala

```diff
@@ -0,0 +1,232 @@
+package com.stripe.brushfire
+package features
+
+import scala.util.Try
+
+import com.twitter.algebird.{ Aggregator, Semigroup, Monoid }
+
+case class DispatchedFeatureEncoding[L](encoder: DispatchedFeatureEncoder)
+extends FeatureEncoding[String, FeatureValue, Map[L, Long]] with Defaults {
```

Comment:
Sorry for the terrible name collisions here, but `FeatureValue` is a type alias for `Dispatched[Double, String, Double, String]`